### DISCUSSION TITLE
Who is using the atlas cluster profile?

### DISCUSSION THREAD

**Discussion by SilasK** [metagenome-atlas/atlas#744] [General]
I made the atlas cluster profile 

[Docs](https://metagenome-atlas.readthedocs.io/en/latest/usage/getting_started.html#cluster-execution)
[Github](https://github.com/metagenome-atlas/clusterprofile)

with the feature that it selects the queues automatically? Please make a reaction to this post..

The pissue is that I want to transition to snakemake v 18, where this cluster profiles are no longer suported... it has great suport for slurm and other scheduller, but the automatic selection of queus is not on the same level.

**Comment by SilasK** [comment]
@Sofie8 @jmtsuji @bioinfogini @alienzj

**Reply by jmtsuji** [reply]
I am not running ATLAS on a cluster, so I do not think the change will affect me. Thanks for asking about this!

**Reply by bioinfogini** [reply]
@SilasK I work on a cluster and tried to use it, but compiling the yaml was an issue (mem information were pretty hard to define) so I dropped it. I believe one can work also without the automatic queues selection 😃 
Looking forward to updates!

**Reply by Sofie8** [reply]
Hi Silas,
I am submitting my jobs via .slurm on the VSC, but I don't use the cluster profile.
Thanks for the note!
